configs/sites/tier1/acorn/packages_intel.yaml

```diff
@@ -31,7 +31,10 @@
       - '@1.2.1 ~mkl'
     py-numpy:
       require::
-      - '@:1.23 ^openblas'
+      - '^[virtuals=lapack,blas] openblas'
+      - '@:1.25'
```

Comment:
Can you plot with matplotlib and this version with intel@2022?

Reply:
Yes